internal/proto/controller/api/services/v1/authtokens_service.proto

```diff
@@ -49,25 +49,25 @@ service AuthTokenService {
 }
 
 message GetAuthTokenRequest {
-  string id = 1;
+  string id = 1;  // @gotags: `class:"public"`
 }
 
 message GetAuthTokenResponse {
   resources.authtokens.v1.AuthToken item = 1;
 }
 
 message ListAuthTokensRequest {
-  string scope_id = 1 [json_name="scope_id"];
-  bool recursive = 20 [json_name="recursive"];
-  string filter = 30 [json_name="filter"];
+  string scope_id = 1 [json_name="scope_id"];  // @gotags: `class:"public"`
+  bool recursive = 20 [json_name="recursive"];  // @gotags: `class:"public"`
+  string filter = 30 [json_name="filter"];  // @gotags: `class:"public"`
 }
 
 message ListAuthTokensResponse {
-  repeated resources.authtokens.v1.AuthToken items = 1;
+  repeated resources.authtokens.v1.AuthToken items = 1;  // @gotags: `class:"public"`
```

Comment:
This doesn't need a tag

Reply:
According to @tmessi we _do_ need it on repeated fields?

Reply:
I'm pretty sure I tried to remove these any my tests started failing with these list not being processed correctly. I can try again though, maybe I messed something up.

Reply:
To me this feels like the same case as a normal message; the fact that it's repeated is beside the point. The tagging will still proceed according to the inner messages. (And if it doesn't, that feels like a problem that needs to be solved.)

Reply:
Will remove this